src/utils/_Popper.js

```diff
@@ -122,7 +122,7 @@ class Popper extends React.Component {
                             ref={ref}
                             style={{ ...style, ...popperProps.style }}
                             // eslint-disable-next-line no-undefined
-                            x-out-of-boundaries={!!outOfBoundaries || undefined}
+                            x-out-of-boundaries={!!outOfBoundaries ? 'true' : undefined}
```

Comment:
Would this work if we just had  `x-out-of-boundaries={!!outOfBoundaries}`?

Reply:
I tried it, and it throws an error: "If you used to conditionally omit it with x-out-of-boundaries={condition && value}, pass x-out-of-boundaries={condition ? value : undefined} instead." It seems  x-out-of-boundaries expects either a string when true or an undefined when false.